### PR TITLE
Orad/joss update 1

### PR DESCRIPTION
**Pull request by odunbar**
## Purpose 

Latest paper version 
[paper.pdf](https://github.com/CliMA/EnsembleKalmanProcesses.jl/files/10035168/paper.pdf)

## Content

- Add sentence to introduction on docs landing page
- Add state of the field RE julia packages into `paper.md`
- Move statement of need above "features" in `paper.md`

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # EnsembleKalmanProcesses
 
-`EnsembleKalmanProcesses.jl` (EKP) is a library of derivative-free Bayesian optimization techniques based on Ensemble Kalman Filters, a well known family of approximate filters used for data assimilation. Currently, the following methods are implemented in the library:
+`EnsembleKalmanProcesses.jl` (EKP) is a library of derivative-free Bayesian optimization techniques based on Ensemble Kalman Filters, a well known family of approximate filters used for data assimilation.The tools of this library can fit parameters found in expensive black-box computer codes without needing adjoints or  derivatives; they can calibrate non-deterministic models, and with noisy data.
```
Comment (ilopezgp) [inline docs/src/index.md:3]:
```suggestion
`EnsembleKalmanProcesses.jl` (EKP) is a library of derivative-free Bayesian optimization techniques based on ensemble Kalman Filters, a well known family of approximate filters used for data assimilation. The tools in this library enable fitting parameters found in expensive black-box computer codes without the need for adjoints or derivatives. This property makes them particularly useful when calibrating non-deterministic models, or when the training data are noisy.
```

**Review by ilopezgp — COMMENTED** [review]
Thanks for working on this @odunbar . I added some edits below, feel free to use them partially if you prefer a different phrasing.

File: paper.md [hunk 2]
```diff
@@ -91,6 +69,34 @@ EnsembleKalmanProcesses.jl aims to provide a flexible and comprehensive solution
...
 
+## State of the field
+
+Several nonlinear optimizers exist in Julia, collected in Optim.jl and JuliaSmoothOptimizers.jl, for example, but these are suited to smooth models with gradient-based approaches. For gradient-free optimization, some tools are collected within packages such as BlackBoxOptim.jl and Metaheuristics.jl, however, although these packages feature a number of ensemble-based approaches, none utilize Kalman-based statistical updates of ensembles, instead relying on heuristic arguments of genetic algorithms and biologically-inspired rules of swarming. A final related class of methods to calibrate black-box computer codes are based on Bayesian inference, such as (Markov Chain) Monte Carlo, implemented in Turing.jl, for example. Such methods are effective but are far more computationally expensive as they provide an entire joint distribution for model parameters, from which the optimum is taken as the summary statistic.
+
+We feel there is a need for more computationally inexpensive, mathematically-grounded ensemble approaches, such as Kalman methods that are provably optimal in simple settings, and have a large literature of extensions to complex problems.
```
[the remark this answers is not in the document]
Comment (odunbar) [reply paper.md]: done

File: paper.md [hunk 3]
```diff
@@ -91,6 +69,34 @@ EnsembleKalmanProcesses.jl aims to provide a flexible and comprehensive solution
 
 4. With scalability enhancements, such as the ones provided by the \texttt{Localizer} structure, it can be applied to high-dimensional problems.
 
+## State of the field
+
+Several nonlinear optimizers exist in Julia, collected in Optim.jl and JuliaSmoothOptimizers.jl, for example, but these are suited to smooth models with gradient-based approaches. For gradient-free optimization, some tools are collected within packages such as BlackBoxOptim.jl and Metaheuristics.jl, however, although these packages feature a number of ensemble-based approaches, none utilize Kalman-based statistical updates of ensembles, instead relying on heuristic arguments of genetic algorithms and biologically-inspired rules of swarming. A final related class of methods to calibrate black-box computer codes are based on Bayesian inference, such as (Markov Chain) Monte Carlo, implemented in Turing.jl, for example. Such methods are effective but are far more computationally expensive as they provide an entire joint distribution for model parameters, from which the optimum is taken as the summary statistic.
```
[the remark this answers is not in the document]
Comment (odunbar) [reply paper.md]: done

[the remark this answers is not in the document]
Comment (odunbar) [reply paper.md]: I'll see what it looks like with all packages in this form - I agree with consistency

File: paper.md [hunk 4]
```diff
@@ -41,63 +41,71 @@ bibliography: paper.bib
...
+The workflow is similar for ensemble Kalman processes. Here, a computer code is run (in parallel) for an ensemble of different values of the parameters that require calibration, producing an ensemble of outputs. This ensemble of outputs is then compared to observed data, and the parameters are updated to a new set of values which reduce the output–data misfit. The computer model is then evaluated for the new ensemble values and the outputs. Optimality of the update is gauranteed for linear models and Gaussian uncertainties, but good performance is observed outside of these settings, see [@Schillings2017]. The process is iterated until a user-defined criterion of convergence is met. Optimal values are selected from statistics of the final ensemble.
```
Comment (ilopezgp) [inline paper.md:48]:
```suggestion
The workflow is similar for ensemble Kalman processes. Here, a computer code is run (in parallel) for an ensemble of different values of the parameters that require calibration, producing an ensemble of outputs. This ensemble of outputs is then compared to observed data, and the parameters are updated to a new set of values which reduce the output–data misfit. The computer model is then evaluated for the new ensemble values and the outputs. Optimality of the update is guaranteed for linear models and Gaussian uncertainties, but good performance is observed outside of these settings, see [@Schillings2017]. The process is iterated until a user-defined criterion of convergence is met. Optimal values are selected from statistics of the final ensemble.
```

**Review by ilopezgp — APPROVED** [review]

File: paper.md (hunk 4 above)
Comment (odunbar) [reply paper.md]: done

**Pull request merged**